web/src/components/takeover-dialog/styles.ts

```diff
@@ -9,7 +9,7 @@ export default createUseStyles(
       position: 'fixed',
       width: '100%',
       height: '100%',
-      zIndex: 100,
+      zIndex: ({ zIndex }) => 101 || zIndex,
```

Comment:
wait a sec... it should be the other way round:
`({ zIndex }) => zIndex || 101 `

side note: the `blah || default` pattern is a bit dangerous because you will still get the default value if zIndex is set to `0`.

Reply:
can we actually do `({zIndex = 101} = {}) => zIndex` or maybe even `({zIndex = 101}) => zIndex`? 

That would prevent the problem with `0`.

Reply:
I'm trying not to use structure `({zIndex = 101}) => zIndex` in general, as sometimes it will be not clear where default value comes from. In this case, it's obvious, but in general, don't like it. I would prefer `({ zIndex }) => zIndex || 101`, which I just pushed

Reply:
in whcih cases is not clear? in longer functions?

Reply:
in js functions in general, especially when function have one param - object